model/Core/Properties/evaluation.md

```diff
@@ -0,0 +1,19 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# evaluation
+
+## Summary
+
+Evaluation is an outcome considering results of a verification. 
+ 
+## Description
+
+Evaluation is the judgement, assessment or opinion regarding the compliance of a verification outcome with the expectations of this verification to meet the verification's acceptance criteria.
+
+Note: An evaluation with a `inconclusive` EvaluationResultType must have a comment on it.
```

Comment:
```suggestion
```

This is the constraint for `EvaluationResult` class, not for the `evaluation` property itselft.